addons/respawn/initSettings.sqf

```diff
@@ -0,0 +1,30 @@
+[
+    QGVAR(savePreDeathGear),
+    "CHECKBOX",
+    [LSTRING(SavePreDeathGear_DisplayName), LSTRING(SavePreDeathGear_Description)],
+    LSTRING(DisplayName),
+    true,
+    true, {
+        if (_this && {getMissionConfigValue ["respawnDelay", 0] == 0}) then {
+            0.01 remoteExec ["setPlayerRespawnTime"];
```

Comment:
Why `remoteExec`? The code block should run on all machines

Reply:
If this code is exec global than this is pointless... i didnt know that,